stock_picking_auto_revert/models/stock_picking.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 ForgeFlow
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, exceptions, _
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    def _check_restrictions(self):
+        # to be implemented by other modules to add restrictions to returns
```

Comment:
What the aim of this ?

Shouldn't it be located in another module ?

Reply:
It was just a hook for other modules to add restrictions on the use of this button. But it was not a good idea. I removed it.